check/check_test.go

```diff
@@ -53,49 +53,54 @@ func TestCheck_Run(t *testing.T) {
 
 func TestGetFirstValidSubCheck(t *testing.T) {
 	type TestCase struct {
-		SubChecks []SubCheck
+		SubChecks []*SubCheck
 		Chosen    *BaseCheck
 		Expected  *BaseCheck
 	}
 
 	testCases := []TestCase{
 		{
-			SubChecks: []SubCheck{
+			SubChecks: []*SubCheck{
 				{
 					BaseCheck{
 						Audit:       "ls /home | grep $USER",
 						Constraints: map[string][]string{"platform": []string{"ubuntu"}},
 						Remediation: "Fake test, check that current user has home directory",
+						auditer:     Audit("ls /home | grep $USER"),
 					},
 				},
 				{
 					BaseCheck{
 						Audit:       "ls /home | grep $USER",
 						Constraints: map[string][]string{"platform": []string{"Fail", "ubuntu", "grub"}},
 						Remediation: "Fake test, check that current user has home directory",
+						auditer:     Audit("ls /home | grep $USER"),
```

Comment:
@igorbeider Sorry I missed this before, do we need both Audit and auditer in these tests, maybe one of these is redundant?